packages/transforms/federation/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "@graphql-mesh/transform-federation",
```

Comment:
maybe `@graphql-mesh/transform-apollo-federation`? same with the YAML config schema.

Reply:
Which YAML config schema do you mean?

Reply:
This one: 
```graphql
extend type Transform {
  federation: FederationTransform
}
```

It should be
```graphql
extend type Transform {
  apolloFederation: ApolloFederationTransform
}
```

Also, please make sure that it works outside the repo. Because `getPackage` will look for `@graphql-mesh/apolloFederation` and this it not the correct package name. I think we need to change the it to snake-case before looking for the package name according to the transform name.

Reply:
I fixed that casing issue in `getPackage` so it is fine but not sure if we have `apollo` prefix ?

Reply:
I fixed casing issue already. It is automatically converted to `apollo-federation` but not sure if we need `apollo` prefix.